test/config/Dockerfiles/contra-env-setup-test-c7/test_contra_env_setup.py

```diff
@@ -199,8 +188,14 @@ def test_jenkins_running(run_info):
         for line in oc_result.splitlines():
             if 'jenkins' in line:
                 route = line.split()[1]
-                request = requests.get('https://%s/login' % route, verify=False)
-                jenkins_running = True if request.status_code == 200 else False
+                requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+                for i in range(12):
```

Comment:
same

Reply:
Same as above, we retry every 10 seconds to see if jenkins login page is available.

Reply:
ack, 120s, thanks